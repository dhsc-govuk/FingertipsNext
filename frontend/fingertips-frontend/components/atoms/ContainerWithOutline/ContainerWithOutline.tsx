import styled from 'styled-components';
import { BorderColour } from '@/lib/styleHelpers/colours';

export const ContainerWithOutline = styled.div`
  @media (min-width: 641px) {
    border: 1px solid ${BorderColour};
    padding: 30px 20px 5px 20px;
  }
`;
