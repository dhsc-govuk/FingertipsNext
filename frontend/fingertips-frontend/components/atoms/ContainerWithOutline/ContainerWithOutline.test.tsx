import { ContainerWithOutline } from '@/components/atoms/ContainerWithOutline/ContainerWithOutline';
import { render } from '@testing-library/react';

describe('ContainerWithOutline', () => {
  it('should render correctly', () => {
    const { container } = render(
      <ContainerWithOutline>Content</ContainerWithOutline>
    );
    expect(container).toMatchSnapshot();
  });
});
