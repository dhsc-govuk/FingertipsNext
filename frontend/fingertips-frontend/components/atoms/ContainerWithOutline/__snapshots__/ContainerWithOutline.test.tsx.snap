// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerWithOutline should render correctly 1`] = `
@media (min-width:641px) {
  .c0 {
    border: 1px solid #BFC1C3;
    padding: 30px 20px 5px 20px;
  }
}

<div>
  <div
    class="c0"
  >
    Content
  </div>
</div>
`;
